llvm/test/CodeGen/AVR/branch-relaxation-long-forward.ll

```diff
@@ -10,7 +10,14 @@
 ; ATTINY85: ldi r24, 0x3
 ; ATTINY85-NEXT: ret
 
-; AVR25: error: out of range branch target (expected an integer in the range -4096 to 4095)
```

Comment:
Lost error test

Reply:
This error is unreachable now, the previous test was overzealous.

Reply:
Yes. This error has been eliminated in the MC layer, and will be reported by GNU  avr-ld.